doc/images/Operator-Maturity-Detailed.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nit: can we change this to lower case file name instead? e.g. `operator-maturity-detailed.svg`

Reply:
done

Reply:
The filename is not renamed, also where is this file used?

Reply:
sorry, file is now renamed - my local filesystem is case insensitive. The SVG is to preserve edit-ability.